recipes/faampy/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "faampy" %}
+{% set version = "0.1.2" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/ncasuk/faampy.git
+    git_tag: {{ version }}
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+    build:
+        - python
+        - python >=2.7,<=3.5
```

Comment:
Possibly `# [py>35]` or `#[py>=36]`? I almost used a skip to do this but decided not to. Can't remember why right now...